crates/bin/pcli/src/command/tx.rs

```diff
@@ -858,13 +867,72 @@ impl TxCmd {
                     .full_viewing_key
                     .ephemeral_address(OsRng, AddressIndex::from(*source));
 
+                let timeout_height = match timeout_height {
+                    Some(h) => h.clone(),
+                    None => {
+                        // look up the height for the counterparty and add 2 days of block time
+                        // (assuming 10 seconds per block) to it
```

Comment:
Why assume 10s per block? Looking at the current Testnet 64, it looks like blocks actually take about 5s:

![block-times](https://github.com/penumbra-zone/penumbra/assets/657862/7d75ee8e-73ee-465e-9114-7c10b793b627)

which I assume is coming from the cometbft value we set here: https://github.com/penumbra-zone/penumbra/blob/v0.64.1/testnets/cometbft_config_template.toml#L413

Reply:
The number applies to counterparty chains, so we can't get it correct, since they all have different values, it'll just be an approximation. See 

https://twitter.com/larry0x/status/1742764484144320997/photo/1

all the cosmos chains have different block times

Reply:
Ahh, of course. That makes a lot more sense!